### ISSUE TITLE
Osmand Tracker alternatives to Telegram 

### ISSUE THREAD

**Issue by Holamak** [osmandapp/OsmAnd#21641]
### Describe the idea

It would be great if people were able to use their preferred secure messaging service like Signal or WhatsApp to share location or use the location sharing services

### Expected behaviour

I would love to share my live location with friends and family, especially for safety. My family and I don't use telegram and having everyone sign up for another messaging platform is not ideal for privacy or security. It would be ideal if I was able to use my own messaging service, especially secure ones like signal.

### Alternatives you've considered

Right now my solution is to use Google maps or share my eta through messages 

### Context

Essentially I want to share my live location and eta with people so that can know when I will be arriving at my destination and see my progress, I also wish to share location so family and friends can find me if I am away. Ideally this would be an open link I can send to people but I understand and appreciate the security features of using a secure messaging app. Right now my only solution is Google maps (the service I am trying to migrate from)

**Comment by nicolai33** [comment]
Would be an interesting feature. But sounds complicated to implement.

**Comment by vshcherb** [comment]
Yes, we plan to have some local solution that won't rely on external accounts or any other services requiring registration

**Comment by Holamak** [comment]
> Yes, we plan to have some local solution that won't rely on external accounts or any other services requiring registration

That is even better then integration into other messaging apps thank-you!
